android/src/main/java/io/sentry/capacitor/SentryCapacitor.java

```diff
@@ -192,6 +192,13 @@ public void captureEnvelope(PluginCall call) {
         try {
             String envelope = call.getString("envelope");
             final String outboxPath = HubAdapter.getInstance().getOptions().getOutboxPath();
+
+            if (outboxPath != null && !outboxPath.trim().isEmpty()) {
```

Comment:
I guess this check is wrong but I can see that Ive written wrongly as well in the comment.
so ideally:

```
if (outboxPath == null || outboxPath.isEmpty() {
  // log and return
}
```

Reply:
Oh damn my mistake I was doing this quick while having dinner and was planning to run it after 😅